### ISSUE TITLE
md-input: disabled state reduces height of elements by 1px

### ISSUE THREAD

**Issue by bluebirdtech** [angular/material#4787]
The following line in input.scss aligns the dotted border, at the expense of reducing the height of the input:
```margin-bottom: -1px; // Shift downward so dotted line is positioned the same as other bottom borders```

Currently working around it with: (As keeping the size correct is more important for us)
```
md-input-container .md-input[disabled], md-select[disabled] .md-select-value {
    margin-bottom: 0px;
}
```

Tried to fix it with:
```
md-input-container .md-input[disabled] {
    padding-bottom: 2px;
}
```

Which works for input, but for textarea it messes up the height calculation, as an extra pixel is added to the element content also, I believe from here in input.js:
```
      function getHeight() {
        var line = node.scrollHeight - node.offsetHeight;
        return node.offsetHeight + (line > 0 ? line : 0);
      }
```

**Comment by pdore-netfore** [comment]
+1

**Comment by crisbeto** [comment]
This one was fixed with #7919.
